doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -306,7 +306,7 @@ In order to properly set up the data replication, the secondary nodes must be ab
 5. Set up SSH between the primary and the secondary (the secondary must be able to access `/etc/xroad` via ssh)
    * Create an SSH keypair for `xroad` user and copy the public key to authorized keys of the primary node
    (`/home/xroad-slave/.ssh/authorized_keys`)
-   > On RHEL 8, 9: generate a new key which is compliant with FIPS-140-2, for example ECDSA with curve nistp256
+    On RHEL 8, 9: generate a new key which is compliant with FIPS-140-2, for example ECDSA with curve nistp256
```

Comment:
The `> ` in front of here is likely meant to be a blockquote: https://www.markdownguide.org/basic-syntax/#blockquotes-1

You can see it's behavior in: https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md#33-secondary-installation

If this causes issues in VitePress, we should look into how to resolve it so that we are able to keep the blockquotes themselves.